analyses/molecular-subtyping-EPN/run-molecular-subtyping-EPN.sh

```diff
@@ -23,6 +23,7 @@ BREAKPOINTS_CNV=../chromosomal-instability/breakpoint-data/cnv_breaks_densities.
 BREAKPOINTS_SV=../chromosomal-instability/breakpoint-data/sv_breaks_densities.tsv
 FOCAL_GENE_CN=../../data/consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz
 GISTIC_SUBFILE_FOCALBYGENE=cnv-consensus-gistic/focal_data_by_genes.txt
+SNVs=../../data/v11/snv-consensus-plus-hotspots.maf.tsv.gz
```

Comment:
```suggestion
SNVs=../../data/snv-consensus-plus-hotspots.maf.tsv.gz
```
We won't want to specify data versions here. The latest version is always symlinked in `data/` and testing data will only be symlinked to `data` and not a specific version, so this is likely why the GitHub action failed.

Reply:
Thanks, suggestion has been committed